vllm/v1/core/kv_cache_manager.py

```diff
@@ -157,23 +176,23 @@ def get_computed_blocks(
         # sharing, `num_computed_tokens` is always a multiple of
         # `block_size`.
         num_computed_tokens = len(computed_blocks) * self.block_size
-        return computed_blocks, num_computed_tokens
+        return KVCacheBlocks(computed_blocks), num_computed_tokens
 
     def allocate_slots(
         self,
         request: Request,
         num_tokens: int,
-        new_computed_blocks: Optional[list[KVCacheBlock]] = None,
+        new_computed_blocks_obj: Optional[KVCacheBlocks] = None,
```

Comment:
Q: Why rename this?

Reply:
Becase it is with type `KVCacheBlocks` and we need to unwrap it to `list[KVCacheBlock]` before using. Prefer to call the object with type `list[KVCacheBlock]` "new_computed_blocks" so I rename this object to "new_computed_blocks_obj"

Reply:
But when would we ever use `list[KVCacheBlock]` directly? The _obj suffix feels a bit awkward to me.

Reply:
`list[KVCacheBlock]` is used inside allocate_slots

Reply:
I thought we kinda decided to blur the distinction when introducing the `KVCacheBlocks `class. I don't understand why this particular confusion is a concern for you.

Reply:
Otherwise I need to change `new_computed_blocks` in this function to another name.

Reply:
I personally prefer that, because I really feel `new_computed_blocks_obj` awkward. But I'm fine if you want to stick to this.

Reply:
OK. I can change it later.